### ISSUE TITLE
[A11Y | Segmented Button]: Not possible to interact on Android tablet with screen reader

### ISSUE THREAD

**Issue by PlutaKatarzyna** [SAP/ui5-webcomponents#11477]
### Bug Description

When Segmented Button is used on Android tablet (Samsung Galaxy Tab S5e) with TalkBack screen reader, it is not possible to activate the button.

### Affected Component

UI5 Segmented Button, UDEx Segmented Button

### Expected Behaviour

It should be possible to activate the button with screen reader enabled.

### Isolated Example

https://staging-srch-is-angular.bfe61ce.stage.kyma.ondemand.com/?isu_page=1 

### Steps to Reproduce

1. Open https://staging-srch-is-angular.bfe61ce.stage.kyma.ondemand.com/?isu_page=1 on Android tablet with the screen reader.
2. Search with a query.
3. Navigate using swipe gesture to the "List view" and "Card view" buttons.
4. Double tap to activate it.
5. Pay attention to the screen reader focus and observe the result.

### Priority

High

### UI5 Web Components Version

1.24.20

### Browser

Chrome

### Operating System

Android 11

### Additional Context

UDEx ticket reported for Segmented Button https://github.tools.sap/sapudex/digital-design-system/issues/1868 .
 UDEx component extends UI5 component.

### Organization

SAP UDEx Components & Tokens

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by unazko** [comment]
Hello @PlutaKatarzyna,

I've tested with TalkBack and VoiceOver for comparison.
The issue is reproducible as described via TalkBack and it is not reproducible via VoiceOver for the segmented button.
The segmented button is implemented according to the specification and represents a `listbox` with `option`s.
Also I've confirmed that the issue is also reproducible via native HTML with the WAI-ARIA samples:
https://www.w3.org/WAI/ARIA/apg/patterns/listbox/examples/listbox-scrollable/

That would mean that the same issue would be there for all `listbox` with `option`s. Currently such DOM structure is
is present for all of the input controls with suggestions.

There is already a chromium issue logged about the same behaviour: https://issues.chromium.org/issues/40260928

I'm closing this incident as this is a TalkBack issue and there is nothing we can do at our side.

Best regards,
Boyan
